server/homebrew.api.js

```diff
@@ -6,13 +6,52 @@ const GoogleActions = require('./googleActions.js');
 const Markdown = require('../shared/naturalcrit/markdown.js');
 const yaml = require('js-yaml');
 const asyncHandler = require('express-async-handler');
+const { nanoid } = require('nanoid');
 
 // const getTopBrews = (cb) => {
 // 	HomebrewModel.find().sort({ views: -1 }).limit(5).exec(function(err, brews) {
 // 		cb(brews);
 // 	});
 // };
 
+const getBrew = (accessType)=>{
+	return async (req, res, next)=>{
+		const { brew } = req;
+
+		if(!brew) {
+			let id = req.params.id, googleId = req.body?.googleId;
+
+			if(id.length > 12) {
+				googleId = id.slice(0, -12);
+				id = id.slice(-12);
+			}
+			let found = await HomebrewModel.get(accessType === 'edit' ? { editId: id } : { shareId: id })
```

Comment:
Can we change `found` to `stub` or something?

Reply:
Sure!